### PR TITLE
Update nfd group members

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: groups/sig-node/groups.yaml [hunk 1]
```diff
@@ -45,7 +45,8 @@ groups:
       - carangog@redhat.com
       - jordanjacobelli04@gmail.com
       - markus.lehtonen@intel.com
-      - zkosic@redhat.com
+      - carangog@redhat.com
```
Comment (ArangoGutierrez) [inline groups/sig-node/groups.yaml:48]:
```suggestion
```

**Review by marquiz — APPROVED** [review]
/lgtm

**Comment by ameukam** [comment]
/assign @dchen1107 @derekwaynecarr 
For approval from the SIG chairs.

**Comment by ArangoGutierrez** [comment]
👋🏽 @dchen1107 @derekwaynecarr

**Comment by ArangoGutierrez** [comment]
👋🏽  @dchen1107 @derekwaynecarr
/woof

**Review by mrunalp — APPROVED** [review]

**Pull request merged**